### ISSUE TITLE
Identify source of bad RSS dates

### ISSUE THREAD

**Issue by waldoj** [openva/rs-video-processor#55]
What’s happening here?

```
Found new video, for 19691231, at: http://archive-media.granicus.com:443/OnDemand/virginia-senate/virginia-senate_f95779cf-18f4-4698-adb5-13b5548ec846.mp4
```

The RSS looks OK:

```xml
<title>Education and Health (SR A) - January 18, 2018 - 8:00 a.m. - Jan 18, 2018</title>
<pubDate>Thu, 18 Jan 2018 05:15:00 -0800</pubDate>
```

My guess is this has something to do with committee videos vs. floor videos.

Figure out the source of this problem, and make sure that the solution involves identifying floor videos as such in the data that's written to SQS.

**Comment by waldoj** [comment]
Ah-ha. In [the RSS feed](http://virginia-senate.granicus.com/VPodcast.php?view_id=3), floor sessions have this element:
```xml
<title>January 18, 2018 - Regular Session - Jan 18, 2018</title>
```
And session videos have this element:
```xml
<title>Finance (Comm Room B)- January 18, 2018 - 4:00 PM - Jan 18, 2018</title>
```
We're breaking up this field by `-`, and using the first to determine the date.

I think the solution is to use everything after the _last_ `-` to determine the date.

**Comment by waldoj** [comment]
Yup—fixed with ccd71f76bcbaac891167d89bbb9a825ba5f8eeba.
